.ci/oci-e2e-has.sh

```diff
@@ -0,0 +1,88 @@
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+command -v e2e-appstudio >/dev/null 2>&1 || { echo "e2e-appstudio bin is not installed. Please install it from: https://github.com/redhat-appstudio/e2e-tests."; exit 1; }
+command -v kubectl >/dev/null 2>&1 || { echo "kubectl is not installed. Aborting."; exit 1; }
+
+if [[ -z "${GITHUB_TOKEN}" ]]; then
+  echo - e "[ERROR] GITHUB_TOKEN env is not set. Aborting."
+fi
+
+if [[ -z "${QUAY_TOKEN}" ]]; then
+  echo - e "[ERROR] QUAY_TOKEN env is not set. Aborting."
+fi
+
+if [[ -z "${HAS_CONTROLLER_IMAGE}" ]]; then
+  echo - e "[ERROR] HAS_CONTROLLER_IMAGE env is not set. Aborting."
+fi
+
+export TEST_BRANCH_ID=$(date +%s)
+export MY_GIT_FORK_REMOTE="qe"
+export MY_GITHUB_ORG="redhat-appstudio-qe"
+export MY_GITHUB_TOKEN="${GITHUB_TOKEN}"
+export WORKSPACE=$(dirname $(dirname $(readlink -f "$0")));
+
+# Available openshift ci environments https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables
+export ARTIFACTS_DIR=${ARTIFACT_DIR:-"/tmp/appstudio"}
+
+# Secrets used by pipelines to push component containers to quay.io
+function createControllerSecrets() {
```

Comment:
Shouldn't we extract these functions to some common place? I guess most of these functions will be the same in other repos, so it would make sense to have them in one place. WDYT?

Reply:
@rhopp Agree! I will create common functions in e2e repository

Reply:
@rhopp Done in https://github.com/redhat-appstudio/e2e-tests/pull/24/files